src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1047,10 +1052,10 @@ def process_blob_delete_batch_parameters(cmd, namespace):
 def _process_blob_batch_container_parameters(cmd, namespace, source=True):
     """Process the container parameters for storage blob batch commands before populating args from environment."""
     if source:
-        container_arg, container_name_arg = 'source', 'source_container_name'
+        container_arg, container_name_arg = 'source', 'container_name'
     else:
         # destination
-        container_arg, container_name_arg = 'destination', 'destination_container_name'
+        container_arg, container_name_arg = 'destination', 'container_name'
```

Comment:
`_process_blob_batch_container_parameters` is used in `upload_batch`, `download_batch` and `delete_batch`, have you confirmed that all of them have changed `source_container_name/destination_container_name` to `container_name`?